marklogic-data-hub-central/ui/src/components/project-page/project-page.test.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {render, cleanup, } from "@testing-library/react";
+import ProjectPage from './project-page';
+import {AuthoritiesContext, AuthoritiesService} from "../../util/authorities";
+import {BrowserRouter as Router} from "react-router-dom";
+
+describe('Update data load settings component', () => {
+
+    afterEach(() => {
+        cleanup();
+    });
+
+    test('should verify project info display', async () => {
+        const authorityService = new AuthoritiesService();
+        authorityService.setAuthorities(['downloadProjectFiles']);
+        const { debug,getByText } = render(<Router><AuthoritiesContext.Provider value={authorityService}><ProjectPage  projectPageVisible={true} setProjectPageVisible={jest.fn()}/>
+        </AuthoritiesContext.Provider></Router>);
+        expect(getByText('Data Hub Version:')).toBeInTheDocument();
+        expect(getByText('MarkLogic Version:')).toBeInTheDocument();
+        expect(getByText('Download Configuration Files')).toBeInTheDocument();
+        expect(getByText('Clear All User Data')).toBeInTheDocument();
+        expect(getByText('Download a zip file containing flow definitions, step definitions and other user artifacts created or modified by Hub Central.')).toBeInTheDocument();
+        expect(getByText('Delete all user data in STAGING,FINAL and JOBS databases.')).toBeInTheDocument();
+        expect(getByText('Clear')).toBeEnabled();
+        expect(getByText('Download')).toBeEnabled();
+    });
+
+    test('should verify project info display', async () => {
```

Comment:
can you pls update the test description to convey what is being tested. In this case "as a user who cannot download"